app/services/live.js

```diff
@@ -30,6 +30,9 @@ export default class LiveService extends Service {
   @globalRef('videoEl') videoEl;
   @tracked peers;
   @tracked isScreenShareOn;
+  @tracked isAnyMavenPresent = '';
+  @tracked isAnyModeratorPresent = '';
```

Comment:
`isAnyMavenPresent` and `isAnyModeratorPresent`
variable starting with "is" mostly indicates that the variable holds a boolean value

Reply:
- Agree with Bhavika here Should it be named to  `mavenRole`?

Reply:
done